### PR TITLE
Allow searching for exact phrases

### LINKED ISSUE #1259: Allow searching for exact phrases vs. AND'ing all search terms
Our Elasticsearch indexer uses AND on all terms entered in a search.  It would be nice if we could also search by phrase, maybe using double-quotes?

I was trying to find a blog post by entering an exact phrase I know is in it, but that returns a ton of other searches.

### PR DESCRIPTION
**Pull request by c3ho**
## Issue This PR Addresses
Fixes #1259 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

Reduces the amount results returned by full-text searching by removing fuzziness. Also allowing users to search by exact phrases.

Adding tests after people give feedback on this.

To test:
`curl http://localhost:3000/query?search="<searchTermHere>"` with a running local backend
feel free to pipe it to a file for verification by appending `> <fileName>.txt` to the end of the above command
or use PostMan
or run local backend + build frontend and use the search Posts function

`searchTermHere` should now support the following:
image

For example if I want to check posts containing the keyword `pandas` OR `c3ho`. I would use:
`curl http://localhost:3000/query?search="pandas+|+c3ho"`

to Test Exact Phrasing:
Linux
`curl http://localhost:3000/query?search='pandas+|+%22c3ho+for+carefully%22'` 

**`%22` = escaped double quotes and `+` = spaces**

Running local back-end + front-end
image

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/backend/utils/indexer.js
Comment (humphd) [inline src/backend/utils/indexer.js]: Does this fix what we discussed in the other PR that Stella is working on?  If so, can you mention that in the other PR?

**Review by humphd — APPROVED** [review]

File: src/backend/utils/indexer.js
Comment (c3ho) [reply src/backend/utils/indexer.js]: I pushed a new commit to remove the above line so she can take care of it in her issue

**Review by manekenpix — APPROVED** [review]

**Comment by c3ho** [comment]
Going to wait on #1221 to go in first before I rebase + merge this PR

**Pull request merged**
